scripts/get_golang_version.sh

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env bash
-# Required GOLANG Version for project specified here
-GOLANG_VERSION="1.14.7"
 
-# Check to make sure that it matches what is specified in go.mod
+# Required GOLANG versions for project specified here.
 
-GOMOD_VERSION=$(awk '/^go[ \t]+[0-9]+\.[0-9]+(\.[0-9]+)?[ \t]*$/{print $2}' go.mod)
-TRIMMED_GOLANG_VERSION=$(echo ${GOLANG_VERSION} |awk -F. '{ print $1 "." $2 }')
+BUILD=1.14.7
+MIN=1.14
+GO_MOD_SUPPORT=1.12
 
-if ! [[ ${GOLANG_VERSION} == "${GOMOD_VERSION}" || ${TRIMMED_GOLANG_VERSION} == "${GOMOD_VERSION}" ]]
+if [ "$1" = all ]
 then
-    echo "go version mismatch, go mod version ${GOMOD_VERSION} does not match required version ${GOLANG_VERSION}"
-    exit 1;
+    echo $BUILD $MIN $GO_MOD_SUPPORT
+elif [ "$1" = dev ]
```

Comment:
it's a bit weird that you're comparing "$1" to dev rather than "dev".

Reply:
I'm not really following.  What's weird about it?

Reply:
I think just that it's not explicitly stating it's a string. It does look a little strange, but am guessing it works.

Reply:
I'm in the habit of not using the double-quotes when there are no shell parameters to expand.

Reply:
I added the double-quotes.